packages/fx-core/src/component/generator/generator.ts

```diff
@@ -53,9 +61,11 @@ export class Generator {
     const replaceMap = ctx.templateVariables;
     const generatorContext: GeneratorContext = {
       name: language ?? commonTemplateName,
-      relativePath: scenario,
+      relativePath: `${scenario}/`,
       destination: destinationPath,
       logProvider: ctx.logProvider,
+      tryLimits: templateDefaultTryLimits,
+      timeoutInMs: templateDefaultTimeoutInMs,
```

Comment:
What about making default timeout 20s? so that we don't need to explicitly specify timeout for templates. So as try limits.